docs/reference/indices/forcemerge.asciidoc

```diff
@@ -10,6 +10,12 @@ This call will block until the merge is complete. If the http connection is
 lost, the request will continue in the background, and any new requests will
 block until the previous force merge is complete.
 
+[WARNING]
+Force merge should only be called against *read-only indices*. Running 
+force merge against a read-write index can cause very large segments to be produced 
+(>5Gb per segment), and the merge policy will never consider it for merging again until 
+it has 75% of deleted docs. This can cause very large segments to remain in the shards.
```

Comment:
I don't think this 75% is correct, it would be shard_size/5GB but I think it's fine to say "until it mostly consists of deleted docs".